packages/manager/src/features/CloudPulse/Alerts/ContextualView/AlertInformationActionTable.tsx

```diff
@@ -88,6 +114,8 @@ export const AlertInformationActionTable = (
   };
   const handleConfirm = React.useCallback(
     (alert: Alert, currentStatus: boolean) => {
+      if (!entityId || !entityName) return;
```

Comment:
```suggestion
      if (entityId === undefined) return;
```

entityId is the source of truth for existence of instance. Also check it for undefined it can give 0 also as false